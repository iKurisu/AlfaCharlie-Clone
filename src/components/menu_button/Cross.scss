.menu-cross {
  width: 20px;
  position: absolute;
  left: calc(50% - 10px);
}

.menu-cross line {
  fill: none;
  stroke: #3a3c48;
  stroke-width: 5;
}
