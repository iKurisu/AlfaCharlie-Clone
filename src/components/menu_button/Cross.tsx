import React from "react";
import "./Cross.scss";

const Cross = (): JSX.Element => (
  <svg className="menu-cross" x="0px" y="0px" viewBox="0 0 59 59">
    <line x1="6.7" y1="6.5" x2="53" y2="53" />
    <line x1="6.7" y1="53" x2="53" y2="6.5" />
  </svg>
);

export default Cross;
