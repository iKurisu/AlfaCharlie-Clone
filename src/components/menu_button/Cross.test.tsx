import React from "react";
import renderer from "react-test-renderer";
import Cross from "./Cross";

it("renders correctly", (): void => {
  const cross: renderer.ReactTestRenderer = renderer.create(<Cross />);

  expect(cross).toMatchSnapshot();
});
