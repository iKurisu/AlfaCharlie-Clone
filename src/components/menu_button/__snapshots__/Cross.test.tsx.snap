// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<svg
  className="menu-cross"
  viewBox="0 0 59 59"
  x="0px"
  y="0px"
>
  <line
    x1="6.7"
    x2="53"
    y1="6.5"
    y2="53"
  />
  <line
    x1="6.7"
    x2="53"
    y1="53"
    y2="6.5"
  />
</svg>
`;
